### PR TITLE
feat(edit-message): Edit image messages

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #10072

[relevant STATUSGO PR](https://github.com/status-im/status-go/pull/3393)

### What does the PR do

Adds possibility to change a text for image messages

### Affected areas

Edit messages

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### CODE REVIEW

**Review by MishkaRogachev — APPROVED** [review]

File: ui/StatusQ/src/StatusQ/Components/StatusMessage.qml [hunk 1]
```diff
@@ -318,31 +318,32 @@ Control {
                                 spacing: 9
                                 Repeater {
                                     model: root.messageDetails.albumCount
-                                    Loader {
+
+                                    delegate: Loader {
```
Comment (caybro) [inline ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:322]: I see what you did there ;) Was about to report the bug :D

**Review by caybro — APPROVED** [review]
LGTM!

**Review by jrainville — APPROVED** [review]

File: ui/StatusQ/src/StatusQ/Components/StatusMessage.qml (hunk 1 above)
Comment (borismelnik) [reply ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:322]: Yeah I waste a lot of time to understand why its doesn't work or showing just one image for all images in album :D

Comment (caybro) [reply ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:322]: Right, I was only seeing the placeholders, never the actual images

**Comment by caybro** [comment]
@borismelnik can we merge this? :)

**Comment by caybro** [comment]
@borismelnik ping? I have a separate diff that fixes just displaying the images-as-album, in case this PR takes longer than expected

**Comment by borismelnik** [comment]
> @borismelnik ping? I have a separate diff that fixes just displaying the images-as-album, in case this PR takes longer than expected

This PR includes edit message functionality for images (which is some kind of feature). It will be done today. But if you need the changes RIGHT NOW because of some reasons we can merge it.

**Comment by caybro** [comment]
> > @borismelnik ping? I have a separate diff that fixes just displaying the images-as-album, in case this PR takes longer than expected
> 
> This PR includes edit message functionality for images (which is some kind of feature). It will be done today. But if you need the changes RIGHT NOW because of some reasons we can merge it.

It's ok, I know this PR contains more than just the bugfix

**Pull request merged**